saleor/settings.py

```diff
@@ -97,7 +97,7 @@
         'string_if_invalid': '<< MISSING VARIABLE "%s" >>' if DEBUG else ''}}]
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = os.environ.get('SECRET_KEY')
+SECRET_KEY = os.environ.get('SECRET_KEY', 'secret-key')
```

Comment:
This could be very dangerous. Everyone should explicitly set `SECRET_KEY` otherwise the majority of users will have the same key.